orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/utils/HealthHelper.groovy

```diff
@@ -112,4 +112,38 @@ class HealthHelper {
     boolean noneAreDown = !healths.any { Map health -> health.state == 'Down' }
     return someAreUp && noneAreDown
   }
+
+  static HealthCountSnapshot getHealthCountSnapshot(List<Map> instances, Collection<String> interestingHealthProviderNames) {
+    HealthCountSnapshot snapshot = new HealthCountSnapshot()
+    instances.each { instance ->
+      List<Map> healths = filterHealths(instance, interestingHealthProviderNames)
+      if (areSomeUpConsideringPlatformHealth(healths, interestingHealthProviderNames, healths.any { Map health -> health.state == 'Up' } )) {
```

Comment:
The comment on `areSomeUpConsideringPlatformHealth()` indicates that it's only intended to be called from one specific method. That comment should probably be updated now.